activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -1,4 +1,7 @@
 module ActiveSupport
+  # These load hooks are how Rails communicates that a component of Rails like
+  # ActionView has fully loaded and that the component can now be extended.
```

Comment:
When you talk about the framework it is Action View. When you talk about the constant it is `ActionView`. You are talking about the framework, so it has a space after the Action.